faq.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Frequently Asked Questions (FAQ)
+---
+
+- this list will be replaced by the Table of Contents
+{:toc}
+
+## Installation
+
+### How do I install a recent version of Cockpit on Debian or Ubuntu?
+
+While Debian and Ubuntu both ship with access to Cockpit in their main repositories, the version in each might be older than the latest available. This is due to the nature of "stable" and "long term support" distributions.
+
+There's often a newer version of Cockpit available for both Debian and Ubuntu, in an extra "backports" repository. We recommend using this newer version, if you're able to do so.
+
+Please read the installation instructions for the appropriate distribution:
+- [Debian installation instructions](running.html#debian)
+- [Ubuntu installation instructions](running.html#ubuntu)
+
+### How do I use Cockpit with a proxy?
+
+Cockpit can be used from behind a proxy.
+
+Read one of the following documents:
+
+- [Proxying Cockpit over NGINX](external/wiki/Proxying-Cockpit-over-NGINX.html)
+- [Proxying Cockpit over Apache with LetsEncrypt](external/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt.html)
+
+## Development
+
+### How do I build an app for Cockpit?
+
+- [Starter Kiit](blog/cockpit-starter-kit.html)
+- [Make your Cockpit page easily installable](blog/making-a-cockpit-application.html)
+- [Additional developer-related resourcesare on the Contributing page](external/wiki/Contributing.html)
+
+## Troubleshooting
+
+### Login
+
+#### Blank white page after login
+
+##### Blank page with a Proxy
+
+You may be using Cockpit with an improperly set up proxy.
+
+###### Solutions
+{:.no_toc}
+
+- [Proxying Cockpit over NGINX](external/wiki/Proxying-Cockpit-over-NGINX.html)
+- [Proxying Cockpit over Apache with LetsEncrypt](external/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt.html)
+
+##### Blank page using Safari
+
+Older versions of Safari on iPhones, iPads, and Safari on macOS on M1 (ARM) machines have a problem with encrypted WebSocket.
+
+###### Solution
+{:.no_toc}
+
+Newer updates of Safari no longer have this issue, but some older [devices may need to use a workaround](running/safari.html).
+
+### Software update
+
+#### Error message about "OSTree"
+
+You're seeing "OSTree is not available on this system" or "Unable to communicate with OSTree" instead of a software update page.
+
+In all liklihood, this means you have `cockpit-ostree` installed on a system that does not use `rpm-ostree` to manage its packages. You should be using Cockpit's built-in PackageKit-based software updater, not `cockpit-ostree`.
```

Comment:
lik*e*lihood

.. should be using Cockpit's PackageKit-based software updater in `cockpit-packagekit`, not ...

Reply:
```suggestion
In all likelihood, this means you have `cockpit-ostree` installed on a system that does not use `rpm-ostree` to manage its packages. You should be using Cockpit's PackageKit-based software updater in `cockpit-packagekit`, not `cockpit-ostree`.
```